skills/dpdata-cli/README.md

```
@@ -0,0 +1,66 @@
+# dpdata CLI Agent Skill
+
+An Agent Skill for using [dpdata](https://github.com/deepmodeling/dpdata) CLI to convert and manipulate atomic simulation data formats.
+
+## Installation
+
+To install this skill, provide the skill's GitHub URL to your agent (e.g., OpenClaw):
+
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)